cmd/helm/testdata/output/get-metadata.txt

```diff
@@ -2,6 +2,8 @@ NAME: thomas-guide
 CHART: foo
 VERSION: 0.1.0-beta.1
 APP_VERSION: 1.0
+ANNOTATIONS: category=web-apps,supported=true
```

Comment:
is this order deterministic?

Reply:
Yes it is. The slice is sorted in the [FormatLabels](https://github.com/kubernetes/apimachinery/blob/v0.31.0/pkg/labels/labels.go#L87) function from k8s.io/apimachinery.

Ref: https://github.com/kubernetes/apimachinery/blob/v0.31.0/pkg/labels/labels.go#L46

The DEPENDENCIES list is sorted in this code change, in the FormattedDepNames method.